packages/ui/stories/organisms/ToggleButton.stories.tsx

```diff
@@ -0,0 +1,75 @@
+/*
+ * Axelor Business Solutions
+ *
+ * Copyright (C) 2023 Axelor (<http://axelor.com>).
+ *
+ * This program is free software: you can redistribute it and/or  modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React from 'react';
+import {View, StyleSheet} from 'react-native';
+import {storiesOf} from '@storybook/react-native';
+import {ToggleButton} from '../../src/components/organisms';
+import {lightTheme} from '../../src/theme';
+
+storiesOf('ui/organisms/ToggleButton', module).add(
+  'Default',
+  args => {
+    return (
+      <View style={styles.container}>
```

Comment:
Can you remove the wrapping view please, else V will have conflicts on his ticket 🥲